### DISCUSSION TITLE
Meeting Won't Publish

### DISCUSSION THREAD

**Discussion by Mrchrisv** [code4recovery/12-step-meeting-list#637] [Q&A] [answered]
Our websiite is area15district3aa.org and we're having an issue with publishing a new meeting.  TSML will take the new meeting entry information and the allow it to be published as long as the location information is left blank.  Once any location and/or region (either new or from the saved list) is entered into those respective fields, the publish button gets grayed out.

**Comment by brianw-area24** [comment]
Are you still having issues? I'm not sure what's going on here, maybe it would help us if you can upload a screen shot of the location information as you're trying to enter them. 

Also, one additional tip, if you are typing in an address, when finished, hit the tab key. This should may help make that publish button active.

Brian W

**Comment by Mrchrisv** [comment]
Yes, I am still having the issue.  I've attached screenshots, but the issue is as soon as anything is entered into the "Address" field of the "Location Information" dialogue box, the "Publish" button becomes inactive.  Using the tab button to navigate makes no difference; the Publish button still becomes inactive.  Also, having the "Location" or the "Region" fields of the "Location Information" populated Does Not have the same effect; just the when anything is entered into the "Address" field.

Our TSML is version 3.12.2  It's been a while since we've added a new meeting so I'm not sure when this issue began.

Thanks in advance for any help you can be in resolving this issue.

Chris V.
District 3, Area 15 Web Chairperson
863.236.8339
www.area15district3aa.org<http://www.area15district3aa.org>



________________________________

**Reply by kiyote33** [reply]
Hi Chris,
Please provide a bit more information to help us troubleshoot the issue you are having. 
What is the WordPress theme you are using on the www.area15district3aa.org site? 
What is the site's WordPress version?
What web browser/version are you using when the problem occurs?
Hopefully we can then replicate your problem and suggest a solution.

**Comment by Mrchrisv** [comment]
Hey there!

If I understand the question correctly, I am not using a packaged  
WordPress theme, but the pages are created using Avada Version 7.2.1
The WordPress version is 5.6.7 and the browser used to access/edit the  
site is Edge

When I look any of the other meetings created in TSML they don't seem  
to have an issue, just ones were trying to create from scratch.

Thanks for sticking with it!

Chris

Quoting kiyote33 ***@***.***>:

**Comment by kiyote33** [comment, marked as answer]
Hi Chris,
Thanks for your response. I am partially able to replicate your issue in my sandbox, but as soon as I completely enter a valid address the Publish button becomes active or enabled in every one of my tests. My testbox is running the latest WordPress version (5.8.3) and I have the same results when utilizing several different themes (i.e...Twenty Twenty One, Twenty Nineteen, Astra, Kadence). An example of a test address used: **122 Main St, Jackson, CA 95642, USA**.  Can you confirm that the issue persists for you when this address is entered? I'm wondering if maybe the addresses you are entering might not be passing the address validity tests done in the background.

Some other thoughts/ideas you might try: If the inactive Publish button continues to be a problem for you, could you run some tests after **updating your WordPress version** to the latest. If the problem persists, could you verify that the issue continues with any of the above mentioned **themes**? If still a problem, try **disabling any installed plugins** (one-at-a-time), to see if one of them is affecting our plugin. Please advise on the results of this testing.  Thx...
